src/pages/rental-details/rental-details.ts

```diff
@@ -33,8 +33,11 @@ export class RentalDetailsPage {
   ngOnInit() {
     this.items = this.itemsService.getItems();
 
+    // We need to offset the date by the difference between the user's timezone
+    // and UTC 0 to have an accurate date
     let tomorrow = new Date();
-    tomorrow.setDate((new Date()).getDate() + 1);
+    const offset = tomorrow.getTimezoneOffset() / (60 * 24);
+    tomorrow.setDate((new Date()).getDate() - offset + 1);
```

Comment:
Instead of instantiating a `new Date()` for the `.getDate()`, I would reuse `tomorrow`:
```JavaScript
tomorrow.setDate(tomorrow.getDate() - offset + 1);
```